### ISSUE TITLE
Tests not generated for paths where variable is passed between parenthesis (e.g. '/companies({id})') ? 

### ISSUE THREAD

**Issue by ralucapredacegeka** [apideck-libraries/portman#445]
Hi,

We are trying to generate tests for our D365 Business Central APIs (just the contract tests for now). 
However, the tests are not generated for paths where variables are provided between between parenthesis: like 'GET https://{businesscentralPrefix}/api/v2.0/companies({id})/contacts({id})'
The following targeting is used in the config:
```json
"contractTests": [
      {
        "openApiOperation": "*::/*",
        "statusSuccess": {
          "enabled": true
        }
      }

```
Is there a way to fix this?

**Comment by thim81** [comment]
@ralucapredacegeka 
That sounds like it is a bug. 
Is it possible to share your openapi.yml or a minimal example that we can use to reproduce the bug?

**Comment by ralucapredacegeka** [comment]
I have modified the https://specs.apideck.com/crm.yml to repro the issue (replaced /{id} with ({id}) )
You can find it in the attached zip together with the config file we used and resulting crm.postman.json
[crm.zip](https://github.com/apideck-libraries/portman/files/10392268/crm.zip)

Please let me know if you need additional details.

**Comment by thim81** [comment]
hi @ralucapredacegeka 

We have a PR in the making, where the result in Postman looks like this:
<img width="656" alt="2023-01-14 at 18 10 38@2x" src="https://user-images.githubusercontent.com/952446/212485841-33c5c5a0-3d62-4c32-8d5e-792ee49d55cb.png">

That is what you would expect?

**Comment by ralucapredacegeka** [comment]
Hi @thim81 ,

Yes, it looks good, thank you!

**Comment by thim81** [comment]
hi @ralucapredacegeka , 
The PR is merged, so the fix should be part of the next release.

**Comment by thim81** [comment]
hi @ralucapredacegeka 

We just released Portman 1.21.0, which includes the fix that solves this issue.
Thanks for taking the time to report the issue. Keep us posted if you run into other issues.
